core/block.js

```diff
@@ -1217,15 +1217,30 @@ Blockly.Block.prototype.getInputsInline = function() {
 /**
  * Set whether the block is disabled or not.
  * @param {boolean} disabled True if disabled.
+ * @deprecated April 2018
  */
 Blockly.Block.prototype.setDisabled = function(disabled) {
+  console.warn('Deprecated call to Blockly.Block.prototype.setDisabled, ' +
+               'use Blockly.Block.prototype.setEnabled instead.');
   if (this.disabled != disabled) {
     Blockly.Events.fire(new Blockly.Events.BlockChange(
         this, 'disabled', null, this.disabled, disabled));
     this.disabled = disabled;
   }
 };
 
+/**
+ * Set whether the block is enabled or not.
+ * @param {boolean} enabled True if enabled.
+ */
+Blockly.Block.prototype.setEnabled = function(enabled) {
```

Comment:
You'll also want to add an isEnabled() so developers don't have to do the inversion on block.disabled to match it up with the API and add @ private to block.disabled (with a note that isEnabled() should be called instead) so we can start encouraging the use of the method calls over the property.